app/elements/coder/challenge/tasks/endpoints/base-components/behaviors/challenge-leaderboard-behavior.html

```diff
@@ -0,0 +1,27 @@
+<script>
+LeaderboardBehavior = {
+	selected: function(){
+		var self = this;
+		self._refresh();
+		this.intervalId = setInterval(function(){
+			self._refresh();
+		}, 5000);
```

Comment:
![](http://vignette2.wikia.nocookie.net/glee/images/e/eb/What-kind-of-sorcery-is-this.jpg/revision/latest?cb=20130422032527)